offload/unittests/OffloadAPI/common/Environment.cpp

```diff
@@ -129,6 +129,9 @@ const std::vector<TestEnvironment::Device> &TestEnvironment::getDevices() {
     }
   }
 
+  if (Devices.size() <= 1)
```

Comment:
I don't really know the code but the `<= 1` looks suspicious here, especially given it's already skipping `OL_PLATFORM_BACKEND_HOST` above. Are you sure it shouldn't be `< 1`? If indeed that is the case, could you add an explanatory comment?